### DISCUSSION TITLE
Mark as read using CLI or API?

### DISCUSSION THREAD

**Discussion by mariomaz87** [wallabag/wallabag#7456] [Q&A]
Hi, 
I have a Calibre recipe that generates an ebook of my unread rss feed from Wallabag. I trigger this recipe using SSH. 
Is there a CLI command I can use to mark all the articles as read in Wallabag? Or any other command I can trigger without having to open Wallabag app itself. Thanks!

**Comment by j0k3r** [comment]
No we don't have such CLI command. But you can mark as read using the api: https://app.wallabag.it/api/doc/#operations-Entries-patch_api_entries__entry_
